Cargo.toml

```diff
@@ -6,9 +6,24 @@ license = "MIT"
 description = "An indentation-based superset of LaTeX"
 readme = "README.md"
 keywords = ["latex"]
+repository = "https://github.com/mp4096/indentex"
 publish = false
 
 
+[package.metadata.deb]
+maintainer = "Mikhail Pak <mikhail.pak@tum.de>"
+copyright = "2016-2017, Mikhail Pak <mikhail.pak@tum.de>, Hans Kirchner <h.kirchner@tum.de>"
+license_file = ["LICENSE.md", "4"]
```

Comment:
`0____0` I didn't read the docs and I was 100% sure that this is the numerical permission for the license file...